.gitignore

```
@@ -0,0 +1 @@
+node_modules
```

We usually add a new line after the end of files. it's called EOF
```suggestion
node_modules

```

---

###